frontend/src/app/animal/animal.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, OnInit } from '@angular/core';
+import { ApiService } from '../api.service';
+
+@Component({
+  selector: 'app-animal',
+  templateUrl: './animal.component.html',
+  styleUrls: ['./animal.component.scss']
+})
+export class AnimalComponent implements OnInit {
+  public animalData;
+  constructor(private apiService: ApiService) { }
+
+  ngOnInit() {
+    this.apiService.getAll('animals').subscribe(
```

Comment:
Animal list from backend must be obtained in `AnimalListComponent`, `AnimalComponent` should contain info about only one animal